roles/provision-event-flow-apb/tasks/provision-manager.yml

```diff
@@ -59,7 +59,7 @@
 
 - name: Add CRD for Flows
   when: flows_result.rc != 0
-  command: "kubectl create -f https://raw.githubusercontent.com/project-streamzi/cloudevent-flow/master/manager/src/main/resources/flow-crd.yml"
+  command: "kubectl create -f https://raw.githubusercontent.com/project-streamzi/cloudeventflow/master/ui/src/main/resources/flow-crd.yml"
```

Comment:
When we move the repo we should rename the project to be 'eventflow'

Reply:
yes, that's planed _after_ this

Reply:
actually _BEFORE_ this - and after the changes on the actual source :grinning: